smart-contracts/test/Lock/behaviors/getBalance.behavior.js

```diff
@@ -0,0 +1,42 @@
+function getBalanceBehavior(options) {
+  describe('Lock / behaviors / getBalance', () => {
+    let accounts, lock, keyPrice
+
+    beforeEach(async () => {
+      ;({ accounts, lock } = options)
```

Comment:
What is the leading `;` for?

Reply:
It's from eslint... but yea it creates a strange syntax. I was trying a few different approaches while putting this together, I'll go back to see if we can change this one a bit.